test/test_helper.rb

```diff
@@ -18,6 +18,8 @@
 # Shim for compatibility with older versions of MiniTest
 MiniTest::Test = MiniTest::Unit::TestCase unless defined?(MiniTest::Test)
 
+RubygemFs.mock!
```

Comment:
What this does is, it changes the backend file system to use a tmp folder. So, theoretically, if only integration tests use the file-system, it is just for it. Only thing is, this is good to happen only once, so thats why I have it at the beginning of the test_helper file.